src/main/java/org/folio/circulation/domain/OpeningPeriod.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.circulation.domain;
+
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.LocalDate;
+import org.joda.time.LocalTime;
+
+public class OpeningPeriod {
+  private static final String OPENING_DAY_KEY = "openingDay";
+  private static final String DATE_KEY = "date";
+
+  private final OpeningDay openingDay;
+  private final LocalDate date;
+
+  public OpeningDay getOpeningDay() {
+    return openingDay;
+  }
+
+  public LocalDate getDate() {
+    return date;
+  }
+
+  public OpeningPeriod(LocalDate date, OpeningDay openingDay) {
+    this.openingDay = openingDay;
+    this.date = date;
+  }
+
+  public OpeningPeriod(JsonObject jsonObject) {
+    this.openingDay = OpeningDay.fromJsonByDefaultKey(jsonObject);
+    String dateProperty = jsonObject.getString(DATE_KEY);
+    this.date = DateTime.parse(dateProperty).toLocalDate();
+  }
+
+  public JsonObject toJson() {
```

Comment:
Similarly, why does this need need to be serialized as JSON?

Reply:
To be returned in a mocked response from `/calendar/periods` in tests.

Reply:
I don't think the production domain code should include methods used for constructing mock responses (which I would prefer worked differently in general)

Reply:
removed

Reply:
Thank you